packages/documentation/src/stories/components/forms/custom-select/custom-select.docs.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as CustomSelectStories from './custom-select.stories.ts';
+import StylesPackageImport from '../../../../shared/styles-package-import.mdx';
+import NgbComponentImport from '../../../../shared/nb-bootstrap/ngb-component-import.mdx';
+import { PostAlert, PostTabs, PostTabHeader, PostTabPanel } from '@swisspost/design-system-components-react';
+import SampleBasicHTML from './custom-select-basic.sample.html?raw';
+import SampleBasicTS from './custom-select-basic.sample.ts?raw';
+import SampleFloatingHTML from './custom-select-floating.sample.html?raw';
+import SampleFloatingTS from './custom-select-floating.sample.ts?raw';
+
+<Meta of={CustomSelectStories} />
+
+<div className="d-flex align-items-center justify-content-between">
+  # Custom-Select
+
+  <a
+  className="btn btn-rg btn-secondary btn-animated"
+  href={`https://archive.design-system.post.ch/#/post-samples/custom-select`}
+  target="_blank"
+  rel="nofollow noopener noreferrer"
+>
+  <span>See archived documentation</span>
+</a>
+</div>
+
+<p className="lead">A custom select for NgBootstrap that uses the ngbDropdown.</p>
+
+<PostAlert type="warning">
+<h4 slot="heading">This component is deprecated</h4>
+<p>It will be removed in the next major version.</p>
+</PostAlert>
```

Comment:
```suggestion
<PostAlert type="warning">
  <h4 slot="heading">This component is deprecated</h4>
  <p>It will be removed in the next major version.</p>
</PostAlert>
```